### PR TITLE
Channel sidebar tabs/content

### PR DESCRIPTION
**Pull request by skddc**
Makes channel sidebar content routable via channel subroutes, and adds header styles and dummy content. Also introduces wide sidebars for tabs with more content.

The sidebars work slightly different on desktop and mobile, as they're optimized for the available screen real estate.

### Desktop

hyperchannel-channel-sidebar-desktop

### Mobile

hyperchannel-channel-sidebar-mobile

### CODE REVIEW

**Comment by skddc** [comment]
Update: you can check this out in production right now: https://hyperchannel.kosmos.org

But please be aware of #197 on small screens.

**Comment by gregkare** [comment]
I checked it on my phone and my laptop. LGTM!

**Comment by skddc** [comment]
group-five

File: app/components/channel-container/template.hbs [hunk 1]
```diff
@@ -9,16 +9,22 @@
       {{this.channel.formattedTopic}}
     </p>
     <nav>
-      <button type="button"
-              {{action "menu" "channel" "toggle"}}
-              class={{if this.headerNavButtonUsersActive "active"}}>
+      <LinkTo @route="space.channel.index" @models={{array space channel}} {{action "menu" "channel" "show"}}>
```
Comment (galfert) [inline app/components/channel-container/template.hbs]: Shouldn't this rather be `[...] {{array this.space this.channel}} [...]`?

Comment (skddc) [reply app/components/channel-container/template.hbs]: Yes, there are hundreds of Octane warnings still. Everyone is invited to help with the upgrade.

**Pull request merged**